apps/dashboard/src/pages/settings.tsx

```diff
@@ -0,0 +1,124 @@
+import { Card } from '@/components/primitives/card';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { OrganizationProfile, UserProfile } from '@clerk/clerk-react';
+import { DashboardLayout } from '../components/dashboard-layout';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { ROUTES } from '@/utils/routes';
+
+export const clerkComponentAppearance = {
+  elements: {
+    navbar: { display: 'none' },
+    navbarMobileMenuRow: { display: 'none !important' },
+    rootBox: {
+      width: '100%',
+      height: '100%',
+    },
+    cardBox: {
+      display: 'block',
+      width: '100%',
+      height: '100%',
+      boxShadow: 'none',
+    },
+
+    pageScrollBox: {
+      padding: '0 !important',
+    },
+    header: {
+      display: 'none',
+    },
+    profileSection: {
+      borderTop: 'none',
+      borderBottom: '1px solid #e0e0e0',
+    },
+    page: {
+      padding: '0 5px',
+    },
+  },
+};
+
+export function SettingsPage() {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const currentTab =
+    location.pathname === ROUTES.SETTINGS ? 'account' : location.pathname.split('/settings/')[1] || 'account';
+
+  const handleTabChange = (value: string) => {
+    switch (value) {
+      case 'account':
+        navigate(ROUTES.SETTINGS_ACCOUNT);
+        break;
+      case 'organization':
+        navigate(ROUTES.SETTINGS_ORGANIZATION);
+        break;
+      case 'team':
+        navigate(ROUTES.SETTINGS_TEAM);
+        break;
+    }
+  };
+
+  return (
+    <DashboardLayout headerStartItems={<h1 className="text-foreground-950">Settings</h1>}>
+      <Tabs value={currentTab} onValueChange={handleTabChange} className="w-full">
+        <TabsList
+          align="center"
+          className="border-border/20 relative mt-4 flex w-full items-end justify-start space-x-2 rounded-none border-b bg-transparent px-1.5 pb-0"
```

Comment:
How about placing it to the right? I feel it looks a bit better.

<img width="1164" alt="Screenshot 2024-12-04 at 16 28 26" src="https://github.com/user-attachments/assets/71ba0287-cf9b-43b1-9a33-f323bac6b292">

Reply:
Evaluated multiple version with Naveen, decided to go with central as overall we thought it looked best until we review this again.